pkg/lib/synthesize/non_inferrable_csv.go

```diff
@@ -0,0 +1,72 @@
+package synthesize
+
+import (
+	"fmt"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apiVer "github.com/operator-framework/api/pkg/lib/version"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+)
+
+type NonInferableCSV struct {
+	Name           string                 `json:"name"`
+	Version        apiVer.OperatorVersion `json:"version"`
+	MinKubeVersion string                 `json:"minKubeVersion"`
+	InstallModes   []v1alpha1.InstallMode `json:"installModes"`
+	Description    string                 `json:"description,omitempty"`
+	DisplayName    string                 `json:"displayName,omitempty"`
+	Keywords       []string               `json:"keywords,omitempty"`
+	Labels         map[string]string      `json:"labels,omitempty" `
+	Links          []v1alpha1.AppLink     `json:"links,omitempty"`
+	Maintainers    []v1alpha1.Maintainer  `json:"maintainers,omitempty"`
+	Maturity       string                 `json:"maturity,omitempty"`
+	Provider       v1alpha1.AppLink       `json:"provider,omitempty"`
+	Selector       *metav1.LabelSelector  `json:"selector,omitempty"`
+	Icon           []v1alpha1.Icon        `json:"icon,omitempty"`
+	Replaces       string                 `json:"replaces,omitempty"`
+	Annotations    map[string]string      `json:"annotations,omitempty"`
+	Descriptions   []Description          `json:"descriptions,omitempty"`
```

Comment:
It's not apparent what `descriptions` means when grokking the metadata file. I strongly suggest we change the name -- at least when serialized -- to something more meaningful; e.g. `apis`.

Reply:
how's `apiDescriptions`?

Reply:
I really want to say description here because the description is a major part of this other than the API gvk.

Reply:
I don't immediately see what the `Descriptions` suffix helps convey to the user.

Reply:
to be clear, I'm talking about the serialization name -- the type can be `APIDescription`

Reply:
well, it is now `apis`